src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml

```diff
@@ -0,0 +1,50 @@
+diagnostics:
+
+  # Data read
+  # ---------
+- data:
+    type: MonDataSpace
+    ctlfile:
+#      - ${data_input_path}/data/time.hirs4_metop-a.ctl
+      - ../data/time.hirs4_metop-a.ctl
+    datasets:
+      - name: experiment
+        satellite: metop-a
+        sensor: hirs4
+        ctlfile:
+#          - ${data_input_path}/diag_amsua_metop-a_ges.2020092200.nc4
+          - ../data/time.hirs4_metop-a.ctl
+        filenames:
+          - ../data/time.hirs4_metop-a.2015051418.ieee_d
```

Comment:
Is there a reason you can't use the absolute path by keeping `${data_input_path}`?

Reply:
That will use ${data_input_path} in the final, non-draft version.

For my own edification, where to I need to set ${data_input_path} when running my tests?  That wasn't immediately clear so I just used this quick hack to get it running.